conda_forge_tick/migrators/core.py

```diff
@@ -287,6 +287,19 @@ def parse_already_pred() -> bool:
         if bad_attr:
             LOGGER.debug("%s: bnad attr" % __name)
 
+        # Check that all upstreams are installable
+        arch_packages = {}
+        # TODO: global variable this somewhere and add aarch and ppc
+        for arch in ['win-64', 'osx-64', 'linux-64']:
+            platform = arch.split('-')[0]
+            if attrs[platform].get('build', {}).get('skip', False):
+                continue
+            req = extract_requirements(attrs[platform])
+            # TODO: maybe throw in test?
+            arch_packages[arch] = set().union(req['host'], req['run'])
```

Comment:
build, host, run, and test are all solved independently, right? I don't mind unioning to get something working. But if we merge it might be good to make an issue.

Reply:
host and run should be solved together, and then when tests are to be run host run and test should be together I think?

(my understanding was that to run the code you needed host+run and then test was on top of that)